tests/python/TestCommand.py

```diff
@@ -41,6 +41,26 @@ def __init__(self, *args, **kwargs):
         self.hostname = "localhost"
         self.port = 55557
 
+        db_up = False
+        attempts = 0
+        while(not db_up):
+            try:
+                db = vdms.vdms()
+                db.connect(self.hostname, self.port)
+                db.disconnect()
+                db_up = True
+                if (attempts > 0):
```

Comment:
This won't print the message if the connection is successful in first try.

Besides, `attempts` will always be greater than zero except the first try, so it may be better to remove the conditional and print the message unconditionally.

Reply:
I think this is intentional. If the first attempt works, no message is printed. If the first attempt fails, it prints a message saying it is retrying, and then prints a message if it is successful.

Reply:
@philiplantz that is correct. Usually there is no problem, so we don't show anything. If there is a problem, we try 10 times and print.